doc/appendix-fr-boi-tva-decl-30-10-30/data-structures/data-structures.md

```diff
@@ -45,9 +45,12 @@ This table describes additional fields of the Charge Items Entry applicable to t
 
 | **Field Name** | **Data Type** | **Default Value Mandatory Field** | **Description**                                                       | **Version** |
 |----------------|---------------|-----------------------------------|-----------------------------------------------------------------------|-------------|
-| `Description`  | `string`      | empty-string<br />mandatory       | Name or description of customary indication or type of other service. | 1.2          |
+| `Description`  | `string`      | mandatory       | Name or description of customary indication or type of other service. | 1.2          |
 | `VATAmount`            | `Decimal`            | 0.0<br />mandatory                           | For French law fulfillment the VAT amount is required. It is used to calculate the net amount in order to avoid rounding errors which are especially likely to appear in row-based net price additions. | 1.2          |
-| `ftChargeItemCaseData` | `string`<br />Max 64k | empty-string<br />mandatory                  | Additional data about the service, currently accepted only in JSON format.                                                                                                           | 1.2          |
+| `ftChargeItemCaseData` | `string`<br />Max 64k | mandatory                  | Additional data about the service, currently accepted only in JSON format.                                                                                                           | 1.2          |
+| `Unit`                 | `String`<br />Max 1k  | mandatory                  | Unit of measurement                                                                                                                                                                  | 0-          |
+                                                                                                    | 0-          |
```

Comment:
This whiteline is causing some troubles with the layout :) 
![image](https://user-images.githubusercontent.com/7176615/94652092-ca5ae980-02f9-11eb-8405-787dffe1a95e.png)